Experts/SymmetryExpert.py

```diff
@@ -60,24 +54,9 @@ def lattice_to_spacegroup_number(lattice):
     """Return the spacegroup number corresponding to the lowest symmetry
     possible for a given Bravais lattice."""
 
-    _lattice_to_spacegroup_number = {
-        "aP": 1,
-        "mP": 3,
-        "mC": 5,
-        "oP": 16,
-        "oC": 20,
-        "oF": 22,
-        "oI": 23,
-        "tP": 75,
-        "tI": 79,
-        "hP": 143,
-        "hR": 146,
-        "cP": 195,
-        "cF": 196,
-        "cI": 197,
-    }
-
-    if lattice not in _lattice_to_spacegroup_number:
+    from LatticeExpert import lattice_to_spacegroup
```

Comment:
can has float import and absolute import?

Reply:
Yarp